astroplan/tests/test_scheduling.py

```diff
@@ -21,70 +22,182 @@
                                      dec=89.26410897 * u.deg), name="Polaris")
 
 apo = Observer.at_site('apo')
-targets = [vega, rigel, polaris]
+targets = [vega, polaris, rigel]
+default_time = Time('2016-02-06 03:00:00')
```

Comment:
Could you call this "tricky_time" or something that indicates that this is a time where float evaluations go bad, that you're testing specifically?